v03/index.js

```diff
@@ -12,8 +12,8 @@ const BinaryHTTPReceiver =
 
 const HTTPUnmarshaller = require("../lib/bindings/http/unmarshaller_0_3.js");
 
-function event() {
-  return new Cloudevent(Spec);
+function newEvent() {
+  return new CloudEvent(Spec);
 }
```

Comment:
If we are exporting the constructor, why even have this function?

Reply:
I didn't attempt to change functionality, just the name in this PR.
I don't want functionality change beyond the name changes.

Reply:
Seems like an arbitrary distinction. I'd argue that making breaking changes as noted above is a change in functionality.